src/menu.cc

```diff
@@ -226,7 +225,7 @@ Menu::Menu() {
           <attribute name='action'>app.source_goto_method</attribute>
         </item>
         <item>
-          <attribute name='label' translatable='yes'>_Rename</attribute>
+          <attribute name='label' translatable='yes'>_Refactor _Rename</attribute>
```

Comment:
I think refactor is a redundant here. One would say "refactor methods" for instance to represent such methods, but "rename" is well known to be a "refactor method".